### ISSUE TITLE
No Rendered Tiles

### ISSUE THREAD

**Issue by gareth-johnstone** [Overv/openstreetmap-tile-server#59]
**Setup**
Cores: 8
RAM: 32gb
SWAP: 120Ggb
Disk Space: ~7tb

**Import**
europe.osm.pbf imported by running the following command:
```
docker run \
    -v /home/pbf/europe-latest.osm.pbf:/data.osm.pbf \
    -v openstreetmap-data:/var/lib/postgresql/10/main \
    overv/openstreetmap-tile-server \
    import
```

This exited after 27 days saying it had managed to import.

**Running the tile server**
I've ran the following command:

```
docker run \
    -p 80:80 \
    -e ALLOW_CORS=1 \
    -e AUTOVACUUM=off \
    -e THREADS=6 \
    -v openstreetmap-data:/var/lib/postgresql/10/main \
    -v openstreetmap-rendered-tiles:/var/lib/mod_tile \
    -d overv/openstreetmap-tile-server \
    run
```

However, when i visit http://145.239.252.99/tile/0/0/0.png it shows a 404 - I'm kinda lost as to where to check next, any suggestions?

**Comment by Overv** [comment]
Could you check the logs of the container to see if your requests are actually handled by it rather than something else?

**Comment by gareth-johnstone** [comment]
What command(s) and where should I check for these logs?

**Comment by Overv** [comment]
To get the container name, run:

    docker ps

And then view the logs with:

    docker logs containername

**Comment by Istador** [comment]
> This exited after 27 days saying it had managed to import.

Whoa, 27 days looks extreme. I thought you'd fixed the slow import, because you closed issue #16.

As [mentioned](https://github.com/Overv/openstreetmap-tile-server/issues/16#issuecomment-457780989) in the other issue, I don't remember the exact duration, but when I imported Europe and Africa together, it only took about ~10-20 hours. (48 cores, 128 GB RAM, 1 TB SSD.)

stevo01 reported in https://github.com/Overv/openstreetmap-tile-server/issues/26#issuecomment-476097846 a runtime of 17.2 hours for importing Europe, that is more similar to your hardware specs. (8 cores, 64 GB RAM, 1 TB NVMe.)

Do you use HDD or SSD/NVMe disk space?

**Comment by gareth-johnstone** [comment]
> > This exited after 27 days saying it had managed to import.
> 
> Whoa, 27 days looks extreme. I thought you'd fixed the slow import, because you closed issue #16.
> 
> As [mentioned](https://github.com/Overv/openstreetmap-tile-server/issues/16#issuecomment-457780989) in the other issue, I don't remember the exact duration, but when I imported Europe and Africa together, it only took about ~10-20 hours. (48 cores, 128 GB RAM, 1 TB SSD.)
> 
> stevo01 reported in [#26 (comment)](https://github.com/Overv/openstreetmap-tile-server/issues/26#issuecomment-476097846) a runtime of 17.2 hours for importing Europe, that is more similar to your hardware specs. (8 cores, 64 GB RAM, 1 TB NVMe.)
> 
> Do you use HDD or SSD/NVMe disk space?

its HDD not SSD - i think thats the biggest issue, they are in RAID 0 though

But yea, i did think it was kinda excessive, but im running the docker image as instructed!



> To get the container name, run:
> 
> ```
> docker ps
> ```
> 
> And then view the logs with:
> 
> ```
> docker logs containername
> ```

Will run this today and report back!
